packages/clay-panel/src/index.tsx

```diff
@@ -201,7 +204,6 @@ function ClayPanel({
 						onExiting={(element) => {
 							element.style.height = '';
 						}}
-						role="tabpanel"
```

Comment:
I was talking to Marcos about this and one of the things we can follow here is to use the w3c accessibility recommendation for accordion which is similar to panel https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/, we can add `role="region"` here instead of `tabpanel`. The other implementations part is ok too.